### PR TITLE
feat: Updates DCO and License Actions

### PR DESCRIPTION
**Pull request by hectorcustodiozup**
Signed-off-by: Hector Custódio <hector.custodio@zup.com.br>

### Related Issues

Update / add opensource security workflows at Beagle iOS repository
https://app.shortcut.com/devrelopensource/story/179041/scorecard-beagle-ios

### Description and Example

Adds validation actions for:
- DCO, 
- License, 
- Dependabot, 
- Horusec.

### Checklist

Please, check if these important points are met using `[x]`:

- [x] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [x] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [x] I am willing to follow-up on review comments in a timely manner.
- [x] I have made the documentation changes or I created an issue explaining how to document this change on [Docs issues](https://github.com/ZupIT/beagle-docs/issues). Please link the issue here:

[PR Guide]: https://github.com/ZupIT/beagle/blob/main/doc/contributing/pull_requests.md

### CODE REVIEW

File: .github/workflows/horusec_pr.yml [hunk 1]
```diff
@@ -1,35 +1,49 @@
...
           parse-json: true
-          
+
       - name: Running Horusec
-        #we need to ignore sample, automatedTests, automated-tests and .gradle because they are not being shipped the Beagle sdk
-        #The two hash was added because we have False Positive on TextInput.swift, we have an enum called PASSWORD, and UrlBuilderDefault.kt, we have a base64 failure but we don't use the base64 on this code
         env:
           HORUSEC_CLI_REPOSITORY_AUTHORIZATION: ${{ env.BEAGLE_CORE_SECURITY_HORUSEC_CLI_REPOSITORY_AUTHORIZATION }}
         run: |
           curl -fsSL https://raw.githubusercontent.com/ZupIT/horusec/master/deployments/scripts/install.sh | bash -s latest
```
Comment (carlossteinzup) [inline .github/workflows/horusec_pr.yml]:
```suggestion
          curl -fsSL https://raw.githubusercontent.com/ZupIT/horusec/main/deployments/scripts/install.sh | bash -s latest
```

**Review by carlossteinzup — APPROVED** [review]

File: .github/workflows/dco-check.yml [hunk 2]
```diff
@@ -14,23 +14,17 @@
...
 on:
   pull_request:
-    branches:
-      - main
+    types: [ opened, synchronize ]
+    branches: [ main, release/* ]
+
 jobs:
-  check:
+  dco-gpg-validator:
```
Comment (hectorcustodiozup) [inline .github/workflows/dco-check.yml]:
```suggestion
  check:
```

File: .github/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+version: 2
+updates:
+  - package-ecosystem: bundler
+    directory: "/"
+    schedule:
+      interval: "weekly"
```
Comment (hectorcustodiozup) [inline .github/dependabot.yml]:
```suggestion
      interval: "monthly"
```

**Review by carlossteinzup — COMMENTED** [review]
Signed-off-by: carlossteinzup <carlos.stein@zup.com.br>

**Pull request merged**